src/components/Button/Button.tsx

```diff
@@ -89,6 +80,37 @@ const Button = ({
       ) : (
         ''
       )}
+    </>
+  );
+
+  const getClass = `
+  ${styles['button_' + category]} 
+  ${styles[size]} 
```

Comment:
This is the work to get the button to be able to be a link as well. Using the href as a flag to make the html an anchor or a button.